src/apiCalls.tsx

```diff
@@ -0,0 +1,14 @@
+import {WeatherLocation} from './model/Weather';
+
+const baseURL = "https://api.openweathermap.org/data/2.5/";
+const apiKey = "bd1b2da0970838af00f1dddfa1582f13";
+
+export async function fetchCityForecast(term: string): Promise<WeatherLocation | undefined>
```

Comment:
Would love to do more research on how a fetch with .thens and async/await might change functionality? Pros/cons of both..